### ISSUE TITLE
Running cargo valgrind shows `error: No runnable target found.`

### ISSUE THREAD

**Issue by changhe3** [jfrimmel/cargo-valgrind#31]
Instead of running my unit tests, I have to manually put them into ./examples to run.

**Comment by jfrimmel** [comment]
Hello, currently the unit tests are not supported. This is due to the the "strange" file name of the unit test executable. That said, it is definitely possible to implement that, and I will try to implement it as soon as possible. Please note, that I'm currently don't have much time to spare on this project.

**Comment by changhe3** [comment]
Thanks for your reply. You should definitely take your time.

**Comment by jfrimmel** [comment]
Hello @changhe3,

It is neary one year ago, but sometimes the priorities are different to what you'd like...

I've just updated the program to (theoretically) support your use-case. If you want to, you can try the new software using the following command:
```bash
cargo install --git https://github.com/jfrimmel/cargo-valgrind --branch custom-runner
```

**Comment by jfrimmel** [comment]
Should be closed by version [2.0.0](https://github.com/jfrimmel/cargo-valgrind/releases/tag/2.0.0). Please update to the new version.

Feel free to reopen this issue, if it is not fixed.
